docs/api-reference/layers/editable-geojson-layer.md

```diff
@@ -216,6 +216,43 @@ Edit handles are the points rendered on a feature to indicate interactive capabi
 
 * Default: `handle => (handle.type === 'existing' ? 5 : 3)`
 
+### Edit Handles (Icons)
```

Comment:
How about the following?

* `editHandleType`: `'point'|'icon'`
* `editHandlePoint...`: are just proxies for the `ScatterplotLayer` props
* `editHandleIcon...`: are just proxies for the [`IconLayer` props](http://deck.gl/#/documentation/deckgl-api-reference/layers/icon-layer?section=properties). So:
  * `editHandleIconAtlas'
  * `editHandleIconMapping`
  * `editHandleIconSizeScale`
  * These accessors would receive an edit handle object as an argument)
  * `editHandleIconGetIcon`
  * `editHandleIconGetSize`
  * `editHandleIconGetColor`
  * `editHandleIconGetAngle`

And I think it'd be cleaner to just have two separate props for getting color, size, and scale rather than have the same prop control two different things, especially if it is mis-named like `getEditHandlePointRadiusScale`.

Reply:
Agreed, this all makes sense.